osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/WatchHighScoresCommand.cs

```diff
@@ -123,9 +123,9 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
                 while (!cancellationToken.IsCancellationRequested)
                 {
                     HighScore[] highScores = (await dbMainQuery.QueryAsync<HighScore>(
-                                                 "SELECT h.*, s.id as new_id FROM osu.score_process_queue "
+                                                 "SELECT q.*, h.*, s.id as new_id FROM osu.score_process_queue q "
                                                  + $"LEFT JOIN {highScoreTable} h USING (score_id) "
-                                                 + $"LEFT JOIN scores s ON h.score_id = s.legacy_score_id AND s.ruleset_id = {RulesetId} "
+                                                 + $"LEFT JOIN scores s ON q.score_id = s.legacy_score_id AND s.ruleset_id = {RulesetId} "
```

Comment:
is this join change meaningful? would like to think that it's not...

Reply:
It actually is, because on deletions the `h` row will be null. Mentioned in https://github.com/ppy/osu-queue-score-statistics/pull/201/commits/f5556a09cd30038eaee89c4357ffa86dfecbda03 commit message.